### PR TITLE
Linkedlist.kt

### PR DESCRIPTION
**Pull request by Apollo9999**
Linkedlist Using Kotlin

### CODE REVIEW

File: Task 3/Linkedlist.kt [hunk 1]
```diff
@@ -0,0 +1,145 @@
+class Node<T>(value: T){
```
Comment (chandrikadeb7) [inline Task 3/Linkedlist.kt:1]: Please mention which kind of Linked List and create a folder and insert the program file inside it.

**Comment by Apollo9999** [comment]
Folder structure created and Readme file of Linkedlist provided.Approve the PR upon review

File: Task 3/Linkedlist.kt (hunk 1 above)
Comment (Apollo9999) [reply Task 3/Linkedlist.kt:1]: Folder structure created and Readme file of Linkedlist provided.Approve the PR upon review

**Comment by Apollo9999** [comment]
Forked the Repo and Created all the Linkedlist files in folder structure .Please let me know again do i need to create a new Pull request

**Comment by chandrikadeb7** [comment]
> Forked the Repo and Created all the Linkedlist files in folder structure .Please let me know again do i need to create a new Pull request

Please mention Singly Linked List as a folder name and insert the code file within it. Please make a new PR.

**Pull request closed without merging**